ContextualMenuDecorator/tests/ContextualMenuDecorator-specs.js

```diff
@@ -10,7 +10,18 @@ const ContextualButton = ContextualMenuDecorator(Button);
 const Root = FloatingLayerDecorator('div');
 const items = new Array(3).fill().map((i, index) => `Option ${index + 1}`);
 
+let MockObserverInstance;
+
 describe('ContextualMenuDecorator Specs', () => {
+	beforeEach(() => {
```

Comment:
It seems we don't need this code anymore.

Reply:
There is code for the existing test in beforeEach. Except that, the resizeObserver implementation has been moved to that test.